m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -1181,10 +1190,13 @@ PROCEDURE GETRECIPSLDATA
        NEWTRANSLDESCRIPTION=LOAN:NICKNAME
        IF NEWTRANSLDESCRIPTION="" THEN
         NEWTRANSLDESCRIPTION=LOAN:DESCRIPTION
+       CHARCHECK=NEWTRANSLDESCRIPTION
```

Comment:
Instead of doing the checking/scrubbing here, can we just do the check where we are outputting the json?

Reply:
@rrobison44 : Removed